aries-site/src/components/home/WhatIs.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+
+import {
+  Box,
+  Grid,
+  Heading,
+  Image,
+  Paragraph,
+  ResponsiveContext,
+} from 'grommet';
+
+const whatIsContent = [
+  {
+    image: '/whatis/base.svg',
+    text: 'Base elements and styles form the rudiments of composition.',
+  },
+  {
+    image: '/whatis/combining.svg',
+    text: 'Combining base elements to make templates.',
+  },
+  {
+    image: '/whatis/tools.svg',
+    text: 'Use tools to help expand and connect elements.',
+  },
+  {
+    image: '/whatis/tailor.svg',
+    text: 'Tailor the code, resources, and tools into one solution.',
+  },
+  {
+    image: '/whatis/app.svg',
+    text: 'Put it all together to make an application.',
+  },
+];
+
+export const WhatIs = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
+      fill
+      pad={{
+        horizontal: size !== 'small' ? 'xlarge' : 'large',
+        top: 'large',
+        bottom: 'medium',
+      }}
+      background="background-front"
+      gap="large"
+    >
+      <Box justify="center" align="center" width="large" alignSelf="center">
+        <Heading>What is the HPE Design System?</Heading>
```

Comment:
```suggestion
        <Heading level={2}>What is the HPE Design System?</Heading>
```
will likely have to adjust size to match desired styling

Reply:
done